docs/how_to/single_node.md

```diff
@@ -32,21 +32,7 @@ curl -X POST http://localhost:7201/api/v1/database/create -d '{
 }'
 ```
 
-**Note:** If you want to create more than one namespace, you should follow the [instructions here](../operational_guide/namespace_configuration.md) and also add the namespace you created to the `local` section of the `m3dbnode-local-etcd.yml` file used in the `docker run` command above with the appropriate aggregation options specified - for more information on our aggregation functionality, check out our [M3Query documentation](query.md). For example:
-
-<!-- TODO: link to aggregation documentation (outside of query.md) -->
-
-```json
-local:
```

Comment:
We should probably have this somewhere, no?

Reply:
Aggregation is hard to talk about without the Prometheus context, so I added a link in the Prometheus integration section to the Query aggregation docs